src/components/AddPropertDialog/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```

Comment:
So we already have a reusable dialog in `src/components/Dialog` but Josh wants to keep that one only for adding records and employees (users) and this one will be used for pretty much everything else.  So please rename this one to `BottomBarDialog` and the old one I will later rename to `UpperBarDialog` so they are easy to differentiate.